### DISCUSSION TITLE
models directory and general questions

### DISCUSSION THREAD

**Discussion by dangerweenie** [gitmylo/audio-webui#240] [Q&A]
Just got this working on Windows and it's great - that said I've been having a hard time finding solid instructions / tutorials.  Coming over from A1111 and ComfyUI so I have a little knowledge and experience.

To start I'm trying to use the Joe Biden voice in the Github repo, but I don't know where to start - is that an .npz file? Or a whole other model?  If you want to do something like a celebrity voice, do you generate the audio in bark and then use an RVC model to convert it?

I downloaded some ,pth, .pt, .npy and .index files from an AI voice repository, but when I put them all in /models/RVC and restart audio-webui, they're still not showing up.  Is there a wiki or directory guide?

I'm in the process of learning how all this stuff works, the relationship between the voice models themselves and the custom voices (it seems to be something akin to using a Stable Diffusion model, and then a LoRA on top of it). been looking through hugging face but I'm not sure what to do with what file and where to put them.  A lot of this stuff is just DYR, of course, but it would be helpful if there were some tutorials on how to do something fun and entry-level like make a greeting with a celebrity voice.

I also can't seem to find any extensions - does anyone know of a repo that hosts them?

TIA for any tips or guidance!

**Comment by gitmylo** [comment]
Make sure the rvc models aren't directly in /models/RVC but in a subdirectory of /models/RVC, for example if you have a model named "example", you could put the .pth and .index files in /models/RVC/example

**Reply by gitmylo** [reply]
As for extensions, there's only one official extension, and I'm not really aware of community made ones, https://github.com/gitmylo/audio-webui-TTS-extension, however, TTS itsself (the library used by the extension) will not be getting updates anymore as Coqui (The creators of TTS) has shut down.
